README.md

```diff
@@ -11,7 +11,9 @@
 - [Copyright](#copyright)
 
 ## Welcome!
-This repository currently hosts a plugin to rerank search results before returning them to the client inline. The current plan is to expand this functionality to allow users to add rewriters, rerankers, and log what those components do. 
+This repository is the home of an evolving project to create a pipeline of transformers to preprocess search queries before information retrieval and post-process results after information retrieval. The first component here is hosts a plugin to re-rank search results before returning them to the client inline. In the coming year, we will add hooks to configure other re-rankers, and allow users to add their own components to the pipeline. Logging will also be a critical part of the pipeline in two ways: 1) Logging information about the search experience (e.g. query, search results returned from the index, search results returned to the OpenSearch client); 2) Logging debug information about the transformers. 
```

Comment:
I'm not sure if `information retrieval` is the correct word to use instead of `search`. It reminds me the classic ways of search like https://en.wikipedia.org/wiki/Information_retrieval, even though we want to provide the capabilities of NLP and other ML techniques.

Reply:
I think it's better to be plain and precise, so I would use "search" instead of "information retrieval". The second sentence has "is hosts" which I'm guessing should be just "is"? Also, is it apparent what "component" means in this context?
```suggestion
This repository is the home of an evolving project that aims to create a pipeline of transformers to preprocess queries before search and post-process results after search. The first component here is a plugin to re-rank search results before returning them to the client inline. In the coming year, we will add hooks to configure other re-rankers and allow users to add their own components to the pipeline. Logging will also be a critical part of the pipeline in two ways: 
1. Logging information about the search experience (e.g. query, search results returned from the index, search results returned to the OpenSearch client) 
1. Logging debug information about the transformers
```

Reply:
I was thinking of information retrieval as the step where we pull results from the actual index. Search is the whole pipeline, but for clarity I can accept this. Thanks.